app/components/project_submissions/item_component.html.erb

```diff
@@ -1,8 +1,8 @@
-<div id="<%= dom_id(project_submission) %>" data-id="<%= project_submission.id %>" data-test-id="submission-item" data-sort-target="item" data-sort-code="<%= sort_code %>">
-  <div class="relative py-6 border-solid border-t border-gray-300 flex flex-col md:flex-row justify-between md:items-center">
+<div id="<%= dom_id(project_submission) %>" data-test-id="submission-item">
+  <div class="relative py-6 flex flex-col md:flex-row justify-between md:items-center">
 
     <div class="flex items-center mb-4 md:mb-0">
-      <%= render ProjectSubmissions::LikeComponent.new(project_submission:, current_users_submission: current_users_submission?) %>
+      <%= render ProjectSubmissions::LikeComponent.new(project_submission:, current_users_submission: false) %>
```

Comment:
Since the LikeComponent has a default value for current_users_submission do we need to specify that it's `false` or are you preferring to be explicit here?

Reply:
Nice spot, I don't mind this being implicit instead.